api_app/analyzers_manager/models.py

```diff
@@ -84,6 +84,8 @@ class MimeTypes(models.TextChoices):
     XPKCS7 = "application/x-pkcs7-signature"
     MIXED = "multipart/mixed"
     X_SHELLSCRIPT = "text/x-shellscript"
+    CRX = "application/x-chrome-extension"
+    JSON = "application/json"
 
```

Comment:
you are changing a model, you need to do a migration for this change too

Reply:
Can I add the mimetypes directly in `0001_initial_squashed.py` migration? @mlodic